src/github/orbit-helpers.js

```diff
@@ -289,12 +240,12 @@ export function _getRepositoryFullName() {
  * @returns Array<String> a 1d array of all repsoitory names, ie ["repo-1", "repo-2"]
  */
 export async function _fetchRepositories() {
-  const { repository_keys } = await chrome.storage.sync.get("repository_keys");
+  const { repository_keys } = await chrome.storage.sync.get({ repository_keys: [] });
```

Comment:
🛑 Breaking change; we give this a default value of `[]` here, but check for `undefined` on line 247

Reply:
Good catch, thanks! Fixed in [47772a7](https://github.com/orbit-love/orbit-browser-extension/pull/38/commits/47772a78bb16b985fcda3b00c57be6cbdd5e7d15).